native/cocos/editor-support/spine-wasm/spine-wasm.cpp

```diff
@@ -4,10 +4,43 @@
 #include "util-function.h"
 #include "wasmSpineExtension.h"
 
+#include <map>
+
 using namespace spine;
 
 namespace {
     HashMap<String, SkeletonData*> skeletonDataMap{};
+
+    static void updateAttachmentVerticesTextureId(SkeletonData* skeletonData, spine::Vector<spine::String>& textureNames, spine::Vector<spine::String>& textureUUIDs) {
+        std::map<std::string, std::string> textureMap;
+        int textureSize = textureNames.size();
+        for (int i = 0; i < textureSize; ++i) {
+            textureMap[textureNames[i].buffer()] = textureUUIDs[i].buffer();
+        }
+
+        auto& skins = skeletonData->getSkins();
+        auto skinSize = skins.size();
+        for (int i = 0; i < skinSize; ++i) {
+            auto* skin = skins[i];
+            auto entries = skin->getAttachments();
+            while (entries.hasNext()) {
+                Skin::AttachmentMap::Entry entry = entries.next();
+                AttachmentVertices* attachmentVertices;
+                auto* attachment = entry._attachment;
+                if (attachment->getRTTI().isExactly(MeshAttachment::rtti)) {
+                    auto* meshAttachment = static_cast<MeshAttachment *>(attachment);
+                    attachmentVertices = static_cast<AttachmentVertices*>(meshAttachment->getRendererObject());
+                } else {
+                    auto* regionAttachment = static_cast<RegionAttachment *>(attachment);
+                    attachmentVertices = static_cast<AttachmentVertices*>(regionAttachment->getRendererObject());
+                }
+                auto* textureName = attachmentVertices->_textureId.buffer();
+                auto iter = textureMap.find(textureName);
+                if (iter == textureMap.end()) continue;
```

Comment:
- What's situation that it will not be able to find the `textureName` in `textureMap`?
- Should deal with it as an error if we could not find the `textureName` ? If so, need to add a warning/error log here.

Reply:
Fixed, add log to ts